.travis.yml

```diff
@@ -66,22 +66,22 @@ jobs:
       python: 3.6
       env: TOXENV=py36-postgres
       services:
-        - postgres
+        - postgresql
         - redis-server
       before_script:
         - psql -U postgres -c "CREATE DATABASE superset;"
         - psql -U postgres -c "CREATE USER postgresuser WITH PASSWORD 'pguserpassword';"
     - language: python
       python: 3.6
       env: TOXENV=pylint
-    - language: python
+    - language: java
```

Comment:
How was this even working before?

Reply:
`jdk_switcher` just happened to be on the python image